packages/venia-ui/i18n/en_US.json

```diff
@@ -94,7 +94,9 @@
     "resetPassword.savedPasswordText": "Your new password has been saved.",
     "resetPassword.successMessage": "Your new password has been saved. Please use this password to sign into your Account.",
     "Returns": "Returns",
-    "Search": "Search",
```

Comment:
I could find no reference to `id={"Search"}` or the like, anywhere in the app.

Yet another reason why we should use the new id format. Much easier to search for uses of it than a plain string...

Reply:
Used to be in `searchTrigger.js`, looks like https://github.com/magento/pwa-studio/pull/2734 took care of it.